.vscode/launch.json

```diff
@@ -14,7 +14,7 @@
         "DEPENDABOT_ORG": "BESTSELLER",
         "DEPENDABOT_CONFIG": "/Users/christian.andersen/BESTSELLER/dependabot-circleci/config.json",
```

Comment:
@ChristianGroentved I dont think it is bestpractice to push your launch.json, that should be in the ignore file :)